static/app/components/modals/widgetViewerModal.tsx

```diff
@@ -192,7 +190,12 @@ function WidgetViewerModal(props: Props) {
             onChange={(option: SelectValue<number>) =>
               router.replace({
                 pathname: location.pathname,
-                query: {...location.query, [WidgetViewerQueryField.QUERY]: option.value},
+                query: {
+                  ...location.query,
+                  [WidgetViewerQueryField.QUERY]: option.value,
+                  [WidgetViewerQueryField.PAGE]: undefined,
+                  [WidgetViewerQueryField.CURSOR]: undefined,
```

Comment:
Usually, we use `cursor` in the URL to define which page we would like the UI to display and it is not clear to me why we need page too

Reply:
The `cursor` for Issues API can sometimes indicate that a previous page exists even when there shouldn't be a previous page. We need to use a separate `page` value to work around this. Both `cursor` and `page` query parameters are used in the Issues page as well